.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: NAME-EXAMINATION UI CI
+
+on:
+  pull_request:
+    types: [assigned, synchronize]
+    paths:
```

Comment:
No paths need in this project. The project layout is different than analytic site.